scripts/grafana/install

```diff
@@ -2,21 +2,37 @@
 
 set -e
 
-GRAFANA_VERSION=5.1.4
+GRAFANA_VERSION=6.0.0
 
 BASE_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
 # Install Grafana dependencies
 sudo apt-get install libfontconfig1 fontconfig-config fonts-dejavu-core ttf-bitstream-vera fonts-liberation sqlite3 -y
 
-# Download and install unofficial build of Grafana for ARM
-URL="https://bintray.com/fg2it/deb/download_file?file_path=main%2Fg%2Fgrafana_${GRAFANA_VERSION}_armhf.deb"
-if uname -m | grep -Eq ^armv6; then
-    # Raspberry Pi 1
-    URL="https://bintray.com/fg2it/deb-rpi-1b/download_file?file_path=main%2Fg%2Fgrafana_${GRAFANA_VERSION}_armhf.deb"
-elif uname -m | grep -Eq ^aarch64; then
-    URL="https://bintray.com/fg2it/deb-arm64/download_file?file_path=main%2Fg%2Fgrafana_${GRAFANA_VERSION}_arm64.deb"
-fi
+ARCH="$(uname -m)" 
+case "$ARCH" in
+  x86_64)
+    URL="https://dl.grafana.com/oss/release/grafana_${GRAFANA_VERSION}_amd64.deb"
+  ;;
+  i386 | i586 | i686 )
+   echo "Cannot install grafana on i386"
+   exit 0
+  ;;
+  armv7l)
+    URL="https://dl.grafana.com/oss/release/grafana_${GRAFANA_VERSION}_armel.deb";
+  ;;
+  armv6l)
+    # Use old grafana - no new one available
```

Comment:
Maybe have a separate `GRAFANA_ARMHF_VERSION` variable to change later?

Reply:
I have not been able to find any other Grafana for armv6l and i dont think it will be updated.
If some one can find it please let me know

Otherwise the options are

- Not have Grafana available
- Keep it static to an old version as it is now :( (at least for now still compatible)
- Build it ourselves (any takes on building it from source :))

We can Create this as an issue if there is interest in it

PS: armv6l = Raspbery Pi 1 and Zero

Reply:
Building a binary ourselves would be the best for compatibility, but there are trust issues, since they won't be confirmed by the devs. Maybe you can contact the devs about this (make a new issue for it in the repo too), and for now we'll leave the old version? I would also some sort of warning with `echo`.